test/Ocelot.UnitTests/Configuration/Validation/FileConfigurationFluentValidatorTests.cs

```diff
@@ -1496,7 +1496,61 @@ public void configuration_is_invalid_when_placeholder_is_used_twice_in_upstream_
                 .Then(x => x.ThenTheResultIsNotValid())
                 .And(x => x.ThenTheErrorMessageAtPositionIs(0, "route /foo/bar/{everything}/{everything} has duplicated placeholder"))
                 .BDDfy();
-        }
+        }
+
+        [Fact]
+        [Trait("Bug", "683")]
+        public void configuration_is_invalid_when_placeholder_is_used_twice_in_downstream_path_template()
+        {
+            this.Given(x => x.GivenAConfiguration(new FileConfiguration
+            {
+                Routes = new List<FileRoute>
+                {
+                    new()
+                    {
+                        DownstreamPathTemplate = "/bar/{everything}/{everything}",
+                        DownstreamScheme = "http",
+                        DownstreamHostAndPorts = new List<FileHostAndPort>
+                        {
+                            new() { Host = "a.b.cd" },
+                        },
+                        UpstreamPathTemplate = "/foo/bar/{everything}",
+                        UpstreamHttpMethod = new List<string> { "Get" },
+                    },
+                },
+            }))
+                .When(x => x.WhenIValidateTheConfiguration())
+                .Then(x => x.ThenTheResultIsNotValid())
+                .And(x => x.ThenTheErrorMessageAtPositionIs(0, "route /bar/{everything}/{everything} has duplicated placeholder"))
+                .BDDfy();
+        }
+
+        [Fact]
```

Comment:
Both tests are very similar.
```suggestion
        [Theory]
```
Why not to merge both cases into one theory?

Reply:
Fixed in commit https://github.com/ThreeMammals/Ocelot/pull/1927/commits/6ef6b216f22ae335a0d2f5b17c80da456fcbf78b ✔️